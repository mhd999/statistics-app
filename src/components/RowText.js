// @flow

import React from 'react';
import styler from 'react-styling';

type Props = {
    text: string,
};

const style = styler `
  TextStyle
    font-size: 12px
    color: #000000
    text-decoration: underline
 `;
const RowText = (props: Props) => (
  <span style={style.TextStyle}>{props.text}</span>
);

export default RowText;