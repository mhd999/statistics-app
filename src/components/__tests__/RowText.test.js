import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import RowText from '../RowText';

describe('<RowText />', () => {
  it('renders a text', () => {
    const component = shallow(
      <RowText text='hello world' />
    );
    const tree = shallowToJson(component);
    expect(tree).toMatchSnapshot();
  });
});
