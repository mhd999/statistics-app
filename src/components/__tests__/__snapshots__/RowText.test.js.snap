exports[`<RowText /> renders a text 1`] = `
<span
  style={
    Object {
      "color": "#000000",
      "fontSize": "12px",
      "textDecoration": "underline",
    }
  }>
  hello world
</span>
`;
